MyAddressBook-final/app/src/main/java/com/example/android/myaddressbook/ContactsActivity.kt

```diff
@@ -353,8 +356,8 @@ class ContactsActivity : AppCompatActivity(), TextWatcher {
         }
 
         internal inner class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-            var nameLabel: TextView = itemView.textview_name
-            var emailLabel: TextView = itemView.textview_email
+            var nameLabel: TextView = itemView.findViewById(R.id.textview_name)
```

Comment:
nit: try to keep consistent between `val a: ExplicitType = findView...` and `val a = findView<ExplicitType>` ?